### PR TITLE
Henry/sto 556 solana docs

### PR DESCRIPTION
**Pull request by HenryMBaldwin**
Reorganized solana example, and added docs to all relevant Solana sub dirs (example, sdk, program).

### CODE REVIEW

File: contracts/solana/README.md [hunk 1]
```diff
@@ -52,3 +88,14 @@ anchor test
 ```
 anchor build
 ```
+#### Initializing contract
+
+```
+ANCHOR_PROVIDER_URL=http://localhost:8899 ANCHOR_WALLET=~/.config/solana/id.json npx ts-node ./app/admin.ts initialize <asset_id> <value>
```
Comment (akawalsky) [inline contracts/solana/README.md:94]: I don't think asset_id and value are required here

Comment (HenryMBaldwin) [reply contracts/solana/README.md:94]: Oh whoops, removing.

File: examples/solana/programs/example/Cargo.toml [hunk 2]
```diff
@@ -18,5 +18,5 @@ idl-build = ["anchor-lang/idl-build"]
 
 [dependencies]
 anchor-lang = "0.30.1"
-stork-sdk = { path = "../../../../sdks/solana/stork-sdk" }
+stork-sdk = { path = "../../../sdks/solana/stork-sdk" }
```
Comment (akawalsky) [inline examples/solana/programs/example/Cargo.toml:21]: Maybe this should use the actual crate?

File: sdks/solana/stork-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Stork Price Feed Accounts
+
+On Solana, Stork price feeds exist as on-chain accounts. These accounts are instances of the [`TemporalNumericValueFeed` account](./src/temporal_numeric_value.rs), and are created and owned by the Stork Oracle contract. These account have an ID which associates them with a specific asset, and a `latest_value` field which stores the latest price update.
+
+## Example
+
+The following is an example of how to use the Stork Solana SDK to consume Stork price updates, available [here](https://github.com/Stork-Oracle/stork-external/tree/main/examples/solana).
+
+```rust
```
Comment (akawalsky) [inline sdks/solana/stork-sdk/README.md:19]: I'm not a huge fan of including so much context in code here as there will likely be drift between these docs and the actual code itself. If you want to do a code snippet, I would limit to much less information and still link the example

**Review by akawalsky — COMMENTED** [review]
A couple comments here

File: examples/solana/programs/example/Cargo.toml (hunk 2 above)
Comment (HenryMBaldwin) [reply examples/solana/programs/example/Cargo.toml]: agreed

File: sdks/solana/stork-sdk/README.md (hunk 3 above)
Comment (HenryMBaldwin) [reply sdks/solana/stork-sdk/README.md]: I slimmed down the snippet a little bit. This readme is all the crates.io landing page for the crate, and having some sort of snippet is pretty standard.

**Review by akawalsky — APPROVED** [review]

**Pull request merged**